datasets/librispeech.py

```diff
@@ -43,9 +44,7 @@ def __init__(self, corpus_key):
             "train-other-500",
         ]
 
-        # this is just needed to have the extraction target available
-        self._out_parent_folder = self.output_path(".", directory=True)
-
+        self._out_archive_folder = self.output_path("LibriSpeech")
```

Comment:
What I meant was not have this "hacky" output at all. See below for move operation.